docs/develop/customize.md

```diff
@@ -22,3 +24,16 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## Change colors
+
+Another common request that you might have is: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
```

Comment:
Ultra-Minor (phrasing):

```suggestion
You might also been wondering: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
```

Reply:
:neckbeard: If going for Pablo's suggestion, edit first few words to "You might also **be** wondering [...]"